specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/informationProtection.json

```diff
@@ -0,0 +1,310 @@
+                "code": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Short constant description of the error"
+                },
+                "message": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Detailed description of the error and resolution suggestions"
+                }
+            }
+        },
+        "Resource": {
+            "type": "object",
+            "description": "Describes an Azure resource.",
+            "properties": {
+                "id": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource Id"
+                },
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource name"
+                },
+                "type": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource type"
+                }
+            },
+            "x-ms-azure-resource": true
+        },
+        "InformationProtectionPolicyList": {
+            "type": "object",
+            "readOnly": true,
+            "description": "Information protection policies response",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "description": "List of information protection policies",
+                    "items": {
+                        "$ref": "#/definitions/InformationProtectionPolicy"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "InformationProtectionPolicy": {
+            "type": "object",
+            "description": "Information protection policy",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Information protection policy data",
+                    "$ref": "#/definitions/InformationProtectionPolicyProperties"
+                }
+            },
+            "allOf": [{
+                    "$ref": "#/definitions/Resource"
+                }
+            ]
+        },
+        "InformationProtectionPolicyProperties": {
+            "type": "object",
+            "description": "describes properties of an information protection policy",
+            "properties": {
+                "lastModifiedUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "readOnly": true,
+                    "description": "Describes the last UTC time the policy was modified"
+                },
+                "sensitivityLabels": {
```

Comment:
Same feedback as https://github.com/Azure/azure-rest-api-specs/pull/3119/files#r191520615. Please change to a dictionary rather than an array.

Reply:
Done